src/app/users/users.component.ts

```diff
@@ -46,10 +48,12 @@ export class UsersComponent implements OnInit {
 		this.userService.getUsers().subscribe(
 			(data) => {
 				this.users = data;
+				this.isLoading = false;
```

Comment:
you should show loading on each start of request and end the loading on response or on error